R/admiral_verify_multiple_templates.R

```diff
@@ -0,0 +1,76 @@
+
+# -------   N ----
+# Step 1, gather 23 datasets from pharamaversadam ---
+
+#  all ds available ---
+library(pharmaverseadam )
+# ...
+
+# 23- get datasets from pharmaverseadam package ("original")
+old = data(package = "pharmaverseadam")
+old = old$results
+old = old[, "Title"]
+old
+
+# but only use 6 ?
+old = list(adae, adcm, adeg, adex, adlb, adsl)
+old[[1]]
+
+
+# step 2 - generate new datasets with admiral templates  ----
+
+# 12-find all templates in `admiral` (when did admiral get loaded?)
+templates = list.files("inst/templates", full.names=TRUE)
```

Comment:
Please use the assign symbol for consistency when not specifying a function argument: `<-` 
```suggestion
templates <- list.files("inst/templates", full.names=TRUE)
```